web/lib/seneca-web-error-handler.js

```diff
@@ -0,0 +1,14 @@
+const Boom = require('boom');
+
+module.exports = (req, reply) => {
+  const resp = req.response;
+  let statusCode = 200;
+  // This only happens because seneca respond with 200 or 500 only
+  // We shouldn't need it in other microservices which are properly using the HTTP standard
+  if (resp && resp.http$) {
+    if (resp.http$.status) statusCode = resp.http$.status;
+    if (resp.http$.redirect) return reply.redirect(resp.http$.redirect);
+    return reply(Boom.boomify(resp.data, { statusCode }));
+  }
+  return reply(resp);
```

Comment:
Some doubts here again (and it's my code again :( )
```
reply - the reply interface which is used to return control back to the framework. To continue normal execution of the request lifecycle, reply.continue() must be called. If the extension type is 'onPostHandler' or 'onPreResponse', a single argument passed to reply.continue() will override the current set response (including all headers) but will not stop the request lifecycle execution. To abort processing and return a response to the client, call reply(value) where value is an error or any other valid response.

```

Reply:
So, you're saying we should use `reply.continue()` instead?

There's nothing left in our chain of handlers at this point though. This is to pick up any errors that result in no reply and send the error to the client, isn't it?